app/src/main/java/com/vinh/dictionary_1/data/model/Word.java

```diff
@@ -114,6 +117,14 @@ public void setWordWithoutDiacritic(String wordWithoutDiacritic) {
         mWordWithoutDiacritic = wordWithoutDiacritic;
     }
 
+    public boolean isBookmarked() {
+        return bookmarked;
+    }
+
+    public void setBookmarked(boolean bookmarked) {
+        bookmarked = bookmarked;
```

Comment:
ERROR: [CORRECTNESS][SA_LOCAL_SELF_ASSIGNMENT_INSTEAD_OF_FIELD] Self assignment of bookmarked rather than assigned to field in com.vinh.dictionary_1.data.model.Word.setBookmarked(boolean)
http://findbugs.sourceforge.net/bugDescriptions.html#SA_LOCAL_SELF_ASSIGNMENT_INSTEAD_OF_FIELD